book/scheduling.md

```diff
@@ -1409,7 +1408,7 @@ class Chrome:
 Event handlers are mostly similar, except that we need to be careful
 to distinguish events that affect the browser chrome from those that
 affect the tab. For example, consider `handle_click`. If the user
-clicked on the browser chrome (meaning `e.y < self.chrome.bottom`), we can
+clicked on the browser chrome, we can
```

Comment:
why this edit?

Reply:
Ah—It breaks across lines in a weird way and seems not critical so rather than rephrase text around it to avoid the weird line break I figured it's easier to delete this text. I think the relevant code used to be "e.y < 100", before we had Chrome do proper font-based layout, and back then the line of code was more cryptic, but I think now it's self-explanatory enough that we don't need to explain it.

Reply:
Not critical IMO